config/routes.rb

```diff
@@ -1,11 +1,8 @@
 Rails.application.routes.draw do
-  root to: "jobs#index"
-
+  root 'home#index'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.